pkg/libmxnet.pkg

```diff
@@ -21,6 +21,7 @@ It has been built with OpenCV and OpenMP support disabled.
 doc_install_dir = '/usr/share/doc/' + VAR.fullname + '/'
 
 ARGS = FUN.mapfiles('./lib', '/usr/lib', libs, append_suffix=False) + [
-    'LICENSE=' + doc_install_dir + 'copyright',
+    'NOTICE=' + doc_install_dir + 'copyright',
+    'LICENSE=' + doc_install_dir,
```

Comment:
You don't need to name a file here?
It looks like `NOTICE` is found in `/usr/share/doc/copyright` but `LICENSE` in `/usr/share/doc/`. But it seems in Debian copyright should contain what is in MXNet's LICENSE.